docs/tutorials/vertical-fl.md

```diff
@@ -35,22 +36,19 @@ python -m pip install -r ./examples/pipelines/requirements.txt
 ```
 
 ## Data provisioning
-The data format for VFL is different from regular FL. That is why each of our examples contains its own script for uploading data that are needed for a given example.
+The data format for VFL is different from regular FL. That is why each of our examples contains its own script for uploading training data.
```

Comment:
I would name it "dataset" or use some more general terminology due to the fact that we split the data into training and validation.